packages/create-nuxt-app/lib/saofile.js

```diff
@@ -103,6 +103,7 @@ module.exports = {
       files: '*',
       filters: {
         '_.eslintrc.js': 'linter.includes("eslint")',
+        '_.eslintignore': 'linter.includes("eslint")',
```

Comment:
Instead of adding another file, can't you add sw.js in gitingore?

Cause there are cases where we use prettier, style lint alongside eslint. This approach would also mean that you have to add `.prettierignore` & `.stylelintignore`

Reply:
Eg: 

Updated ignore - 
https://github.com/vinayakkulkarni/map-my-google-photos/blob/main/.gitignore#L83-L84

Lint scripts - https://github.com/vinayakkulkarni/map-my-google-photos/blob/main/package.json#L11-L18

Reply:
When the files are generated, `sw.*` is included in the `.gitignore` already. Although, eslint still runs through and lints the auto generated service worker codes causing the linting error when developing or building.

Reply:
The error. `@typescript-eslint` is whats emitting the error.

![Screen Shot 2021-08-25 at 3 26 44 AM](https://user-images.githubusercontent.com/20941086/130716531-c97de232-ebb1-417b-9173-64faca56906c.png)

This disappears when `sw.*` is included in the `.eslintignore`